doc/en/components/card.md

```diff
@@ -0,0 +1,307 @@
+
+## Card Example
+
+<code-view style="height: 550px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/layouts/card-overview"
+           alt="$Platform$ Card Example"
+           github-src="layouts/card/overview">
+</code-view>
+
+<div class="divider--half"></div>
+
+## Usage
+
+Cards allow you to easily display content composed of different types of objects or similar objects whose size and supported actions can vary.
+
+### Getting Started
+
+Import `igniteui-webcomponents` inside our **index.ts** file:
+
+```typescript
+// index.ts
+...
+import 'igniteui-webcomponents';
+
+```
+
+Then to represent the demo card template we can add the following code to our **index.html** file.
+
+```html
+<igc-card>
+    <igc-card-media>
+        <img src="https://images.unsplash.com/photo-1518235506717-e1ed3306a89b?ixlib=rb-1.2.1&auto=format&fit=crop&w=640&q=50">
+    </igc-card-media>
+
+    <igc-card-header>
+        <h3 slot="title">New York</h3>
+        <h5 slot="subtitle">City in New York</h5>
+    </igc-card-header>
+
+    <igc-card-content>
+        <p>New York City comprises 5 boroughs sitting where the Hudson River meets the Atlantic Ocean. At its core is Manhattan, a densely populated borough that’s among the world’s major commercial, financial and cultural centers.</p>
+    </igc-card-content>
+
+    <igc-card-actions>
+        <igc-button slot="start">
+            <igc-ripple></igc-ripple>
+            Read more
+        </igc-button>
+        <div slot="end">
+            <igc-icon-button name="twitter">
+                <igc-ripple></igc-ripple>
+            </igc-icon-button>
+            <igc-icon-button name="facebook">
+                <igc-ripple></igc-ripple>
+            </igc-icon-button>
+        </div>
+    </igc-card-actions>
+</igc-card>
+```
+
+You will notice a few things above. First when we want to _tag_ an element as a header title, like the `h3` heading, we place it between the `igc-card-header` tags and set its slot name to `title`. Conversely, if we wanted to make another heading element a `subtitle` we would name its slot `subtitle`.
+
+Any image or video we want to show in the card, we wrap inside the `igc-card-media` tags. The `igc-card-media` allows us to size the content placed inside so that it maintains its aspect ratio while filling the element’s entire content box. If the object's aspect ratio does not match the aspect ratio of its box, then the object will be clipped to fit.
+
+You can place anything inside the `igc-card-content` tags. Usually text goes there.
+
+Finally, the `igc-card-actions` is where you'd place any actionable items, like buttons.
+
+### Media, Thumbs, and Avatars
```

Comment:
there is only a thumbnail example in this section while the media and avatar ones are missing